bindings/go/src/fdb/range.go

```diff
@@ -90,7 +90,10 @@ type ExactRange interface {
 // that the default zero-value of KeyRange specifies an empty range before all
 // keys in the database.
 type KeyRange struct {
-	Begin, End KeyConvertible
+	// the begin (inclusive)
+	Begin,
+	// the end (exclusive)
+	End KeyConvertible
```

Comment:
I like the idea here. Could these be "The (inclusive) beginning of the range" and "The (exclusive) end of the range" or something like that to match the style of the rest of the docs?

Reply:
Resolved 👍